src/nvim/message.c

```diff
@@ -468,17 +505,8 @@ int emsg_not_now(void)
   return FALSE;
 }
 
-/*
- * emsg() - display an error message
- *
- * Rings the bell, if appropriate, and calls message() to do the real work
- * When terminal not initialized (yet) mch_errmsg(..) is used.
- *
- * return TRUE if wait_return not called
- */
-int emsg(const char_u *s_)
+static int emsg_echo(const char *s, bool use_echo)
```

Comment:
`use_echo` and the similar cases should be renamed to e.g. `pretty`. Otherwise it's too confusing, AFAICT it overlaps with other uses of the word "echo" (in this very same file) that do not behave like this.

Reply:
I agree the name is not optimal, but may I ask, what other usages in the same file? I can't find them.

Reply:
the "e" in `emsg`, no?

Reply:
I'm pretty sure that is "error"